pkgs/development/python-modules/ufoLib2/default.nix

```diff
@@ -25,6 +26,8 @@ buildPythonPackage rec {
     fs
   ];
 
+  nativeBuildInputs = [ setuptools-scm ];
```

Comment:
Would be nice to remain consistent regarding the formatting inside each file (one dependency per line for easier `git blame`).